main/res/values-cs/strings.xml

```diff
@@ -777,7 +777,7 @@
     <string name="map_source_osm_cyclosm">CyclOSM</string>
     <string name="map_source_osm_offline">Offline</string>
     <string name="map_source_osm_offline_combined">Kombinované (Offline)</string>
-    <string name="map_source_osm_offline_combined_attribution_details">Kombinovaná (%d map, %d atributů): klikněte pro podrobnosti</string>
+    <string name="map_source_osm_offline_combined_attribution_details">Kombinovaná (%d atributů): klikněte pro podrobnosti</string>
```

Comment:
Your hard-coded translations will be overwritten by crowdin automatically. So if you want to provide these translations, you will have to translate them via the crowdin UI. Or, if you want to make it easy for yourself, you can simply ignore all the translations as outdated ones will be removed automatically...

Reply:
Hi @fm-sys, thanks for the hint. I am aware of that, but I needed to make the modification because lint was complaining due to the mismatch of parameters in the strings vs parameters given in the code (because I reduced the number of paramus from 2 to 1).

So it is fine if those changes are overridden by Crowdin, but I need to make them in order to go through the Android Studio checks (and presumably also through the CI/Crowdin checks although I did not try this out)

Reply:
Then it would be the best to delete such lines.

Reply:
> Then it would be the best to delete such lines.

I still need the translation, it just does not contain 2 parameters an more (just one)

Reply:
We only discuss on the translated strings, not for the english strings.
The translations are only interesting for your coding if it is necessary to use them in your tests.
Only the english strings are interesting otherwise. They are the fallback language if the string not exist in the used language.
The translation work what you have done is useless then. It will be replaced by the crowdin PR. The englich strings are the base for the work at crowdin, nothing else.

Reply:
Strings are irrelevant with newest changes, so I guess discussion can be resolved